ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/UnKeyedTableUtil.java

```diff
@@ -63,6 +65,18 @@ public static Set<String> getAllContentFilePath(Table internalTable) {
       validFilesPath.add(TableFileUtils.getUriPath(entry.getFile().path().toString()));
     }
 
+    TableEntriesScan entriesScan = TableEntriesScan.builder(internalTable)
```

Comment:
- This code seems to duplicate lines between 61-66.
- This method should return all path rather than live ones.